README.md

```diff
@@ -6,6 +6,9 @@ For none ms office files, it works same as `file-type`, but for these ancient fi
 # Install
 `$ npm install file-type-ext`
 
+# Gotchas
+- v2.0.0 must be built using v16.11.10 of node (same version as node that `file-type` v17.1.1 is built with).
```

Comment:
Does this need anything > 16 or that specific node version? It seems like this should need to float

Reply:
I tried with 17.4 and 14.8 and it failed on both of those. Unsure on whether it's that specific node version.

Reply:
I tried on 17.4 and 14.8 and it failed on both. Unsure if it's specifically 16.11 or not. Intended to just walk back from 17 towards 14 until I found one that worked.